packages/backend/src/auto-peering/routes.ts

```diff
@@ -52,7 +52,8 @@ async function acceptPeerRequest(
     ctx.body = {
       error: {
         code: errorCode,
-        message: errorToMessage[peeringDetailsOrError]
+        message: errorToMessage[peeringDetailsOrError],
+        type: peeringDetailsOrError
```

Comment:
Is `type` the right property name for this?

Reply:
What would you propose as an alternative?

I think 
```js
{
    error: {
        code: 400
        message: ...
        type: "UnknownAsset"
    }
}
```
works fine in this case. Similar to stripe docs: https://stripe.com/docs/api/errors#errors-type

Reply:
Yeah, I can't come up with anything better. Let's keep it.

Reply:
type or category make sense
another option is to rename code to status, and use the code yo define the "type" instead